cypress/integration/characteristics.spec.js

```diff
@@ -84,6 +87,9 @@ describe('The Model Plan General Characteristics Form', () => {
     cy.contains('button', 'Next').click();
 
     // Page - /characteristics/involvements
+
+    cy.wait(500);
```

Comment:
Was this test failing without the `wait()`s? I thought Cypress was usually pretty good about waiting for selectors to appear, and 500ms seems like it'd fall in the range of the default timeout (4 seconds, [src](https://docs.cypress.io/guides/references/configuration#Timeouts))

Reply:
Yeah the tests were failing on each page after the GQL refactor, which I assumed could possibly be because those queries were no longer cached on the first page.  I need to look into why it was failing to find the selectors without the wait.

Reply:
Going to work on troubleshooting cypress next sprint 

https://jiraent.cms.gov/browse/EASI-2166